src/components/@core/navigation-menu/light/join-group/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button } from "@chakra-ui/core";
+import useTranslation from "@configs/i18n/useTranslation";
+import { axCheckUserGroupMember, axJoinOpenUserGroup } from "@services/usergroup.service";
+import notification, { NotificationType } from "@utils/notification";
+import { useStoreState } from "easy-peasy";
+import React, { useEffect, useState } from "react";
+
+export default function JoinUserGroup() {
+  const { currentGroup, user } = useStoreState((s) => s);
+  const [isMember, setIsMember] = useState(false);
```

Comment:
by default `undefined` is `falsy`

Reply:
i have changed it to true currently due as  the btn  shows up for few secs if value is falsy